core/controllers/editor.py

```diff
@@ -436,7 +440,7 @@ def get(self, exploration_id):
                 filename, 'text/plain')
         elif output_format == feconf.OUTPUT_FORMAT_JSON:
             self.render_json(exp_services.export_states_to_yaml(
-                exploration_id, version=version, width=width))
+                exploration_id, version=version, width=80))
```

Comment:
The default value of `width` in `export_states_to_yaml` is also 80, not sure whether we need to define the width value here as well! (I'm fine with either.)

Reply:
Also removed. Good catch, thanks!